src/test/rgw/rgw_multi/tests.py

```diff
@@ -397,7 +397,8 @@ def zonegroup_bucket_checkpoint(zonegroup_conns, bucket_name):
                 continue
             log.debug('bucket checkpoint: source=%s target=%s bucket=%s', source_conn.zone.name, target_conn.zone.name, bucket_name)
             zone_bucket_checkpoint(target_conn.zone, source_conn.zone, bucket_name)
-            target_conn.check_bucket_eq(source_conn, bucket_name)
```

Comment:
maybe use a check_bucket_"ge" here is more Intuitive, although both ways can fix the problem

Reply:
it would be difficult to implement a `check_bucket_ge()` for this. and the most important thing to test here is that all zones converge on the exact same result, so i think `check_bucket_eq()` is what we really want